dev/ci/docker_linux/Dockerfile

```diff
@@ -79,19 +82,28 @@ RUN touch ~/.android/repositories.cfg
 ENV ANDROID_SDK_ARCHIVE="${ANDROID_TOOLS_ROOT}/archive"
 RUN wget --progress=dot:giga "${ANDROID_SDK_URL}" -O "${ANDROID_SDK_ARCHIVE}"
 RUN unzip -q -d "${ANDROID_TOOLS_ROOT}" "${ANDROID_SDK_ARCHIVE}"
+RUN ls "${ANDROID_TOOLS_ROOT}"
+
+RUN mkdir "${ANDROID_TOOLS_ROOT}/cmdline-tools/tools"
+RUN ls "${ANDROID_TOOLS_ROOT}/cmdline-tools"
+RUN mv "${ANDROID_TOOLS_ROOT}/cmdline-tools/NOTICE.txt" "${ANDROID_TOOLS_ROOT}/cmdline-tools/tools"
```

Comment:
rather than moving these, can we just change the paths in 96-106 to not have the extra `tools/` dir?

Reply:
The tools dir is needed I believe in order to run this without the android sdk path variable. Using only with cmdline-tools it complained about not being able to find the sdk home.

Reply:
ahh, weird :+1: